v21.2/known-limitations.md

```diff
@@ -609,22 +556,3 @@ If the execution of a [join](joins.html) query exceeds the limit set for memory-
 ### Disk-spilling not supported for some unordered distinct operations
 
 {% include {{ page.version.version }}/known-limitations/unordered-distinct-operations.md %}
-
-### Inverted index scans can't be generated for some statement filters
```

Comment:
@mgartner, please confirm that this limitation can be removed from the 21.2 docs.

Reply:
Confirmed. https://github.com/cockroachdb/cockroach/pull/63048